lms/static/scripts/frontend_apps/components/SubmitGradeForm.js

```diff
@@ -172,6 +172,7 @@ export default function SubmitGradeForm({ disabled = false, student }) {
           ref={inputRef}
           onInput={handleKeyDown}
           type="input"
+          // @ts-expect-error - `defaultValue` is missing from `<input>` prop types.
```

Comment:
This works like `@ts-ignore` except that it will warn if the subsequent statement does _not_ generate an error that gets suppressed.

Reply:
Handy